operator/gitops/argocd/pipeline-service/tekton-results/base/permissions.yaml

```
@@ -0,0 +1,44 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/part-of: tekton-results
+    app.kubernetes.io/version: v0.4.0
+  name: tekton-results-api
```

This name collides with the role from the Tekton Results repo.
```suggestion
  name: tekton-results-db-cred-creator
```